modules/__tests__/BrowserHistory-test.js

```diff
@@ -64,17 +64,11 @@ describe('a browser history', () => {
     });
 
     describe('push with an encoded path string', () => {
-      it('creates a location object with decoded pathname', done => {
+      it('creates a location object with encoded pathname', done => {
         TestSequences.PushEncodedLocation(history, done);
       });
     });
 
-    describe('push with an invalid path string (bad percent-encoding)', () => {
-      it('throws an error', done => {
-        TestSequences.PushInvalidPathname(history, done);
-      });
-    });
-
```

Comment:
This test was no longer valid, since we don't throw an error now we have no decoding. (We assume the pathname provided will be correctly encoded.)